implementations/micrometer-registry-otlp/src/main/java/io/micrometer/registry/otlp/OtlpMeterRegistry.java

```diff
@@ -516,4 +532,25 @@ static Histogram getHistogram(Clock clock, DistributionStatisticConfig distribut
         return NoopHistogram.INSTANCE;
     }
 
+    // VisibleForTesting
+    static double[] getSloWithPositiveInf(DistributionStatisticConfig distributionStatisticConfig) {
+        double[] sloBoundaries = distributionStatisticConfig.getServiceLevelObjectiveBoundaries();
+        if (sloBoundaries == null || sloBoundaries.length == 0) {
+            // When there are no SLO's associated with DistributionStatisticConfig we will
+            // add one with Positive
+            // Infinity. This will make sure we always have POSITIVE_INFINITY, and the
+            // NavigableSet will make sure
+            // duplicates if any will be ignored.
+            return EMPTY_SLO_WITH_POSITIVE_INF;
+        }
+
+        boolean containsPositiveInf = Arrays.stream(sloBoundaries).anyMatch(value -> value == Double.POSITIVE_INFINITY);
```

Comment:
Technically we can also use Double.MAX_VALUE for representing this. But, Double.POSITIVE_INFINITY is already part of the default buckets and hence I chose to prefer POSITIVE_INFINITY over MAX_VALUE.